kolibri/core/assets/src/views/custom-ui-menu/menu-option.vue

```diff
@@ -2,7 +2,6 @@
 
   <li
     class="ui-menu-option"
-    ref="menuOption"
     role="menu-item"
```

Comment:
Role should be `menuitem`, not `menu-item`.

![ubuntu3 start running - oracle vm virtualbox_706](https://user-images.githubusercontent.com/1457929/32972834-9e50b2aa-cbf4-11e7-86e3-8ce8a593fbed.png)